app/(general)/integration/pooltogether-v4/page.tsx

```diff
@@ -0,0 +1,5 @@
+'use client'
+
+export default function PoolTogetherHome() {
+  return <div className="relative flex flex-1"></div>
```

Comment:
this page can be empty, since it only exits to enable the `/integration/pooltogether-v4` page to be accessed, it doens't make sense to add an html tag with tailwind classes here
```suggestion
  return null
```